src/3d/qgstessellator.cpp

```diff
@@ -231,6 +231,14 @@ static void _ringToPoly2tri( const QgsCurve *ring, std::vector<p2t::Point *> &po
   }
 }
 
+
+inline double _round_coord( double x )
```

Comment:
Does inline even do anything anymore? Everything I've read led me to believe it's redundant and the compiler will inline what it wants to regardless of this keyword.

Reply:
> Does inline even do anything anymore?

It does not seem to do much according to this widely accepted answer on SO: https://stackoverflow.com/questions/1759300/when-should-i-write-the-keyword-inline-for-a-function-method#1759575

When I first started learning c++ maybe it still mattered (at least according to the books) and I just kept using it :-)

Reply:
Ok, thanks... was just wondering if you knew some esoteric compiler behavior I wasn't aware of!